include/tiramisu/core.h

```diff
@@ -2483,7 +2482,38 @@ class computation
       */
     computation(std::string name,std::vector<var> iterator_variables, tiramisu::expr e, bool schedule_this_computation, primitive_t t);
 
+    /**
+      * \brief Construct the distribution map of a computation.
+      * Not safe to use. It's currently being implemented.
+      *
+      * A distribution map is a map that describes how the computation is distributed across ranks.
+      * Given the number of available ranks number_of_ranks, the type of the rank rank_type which
+      * is either r_sender or r_receiver, the distribution map will specify for each rank the
+      * iterations it should execute.
```

Comment:
Can you specify also how you do the distribution (or partitioning)?
if you have one dimension you will clearly partition that dimension but if you have two dimensions which one will you partition (the outermost, i.e., leftmost)?